src/deb/control/postrm

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
```

Comment:
Same deal - Please specify with comments which parts of the postrm are Tomcat boilerplate and which parts are unique to Rhino.